.tekton/integration-tests/pipelines/lightspeed-console-e2e-test-pipeline-417.yaml

```diff
@@ -0,0 +1,328 @@
+              export CYPRESS_LOGIN_PASSWORD=$(cat ${PASSWORD_PATH})
+              set +e
+              NO_COLOR=1 npx cypress run
+              err_status=$?
+              echo -n "${err_status}" > /workspace/cypress-exit-code
+              echo "---------------------------------------------"
+              ls ./gui_test_screenshots
+              mv ./gui_test_screenshots /workspace/artifacts/
+              set -e
+              echo "Cypress exit code: ${err_status}"
+              exit ${err_status}
+              echo "---------------------------------------------"
+          - name: gather-cluster-resources
+            onError: continue
+            ref:
+              resolver: git
+              params:
+                - name: url
+                  value: https://github.com/konflux-ci/tekton-integration-catalog
+                - name: revision
+                  value: main
+                - name: pathInRepo
+                  value: stepactions/gather-cluster-resources/0.1/gather-cluster-resources.yaml
+            params:
+              - name: credentials
+                value: "credentials"
+              - name: kubeconfig
+                value: "$(steps.get-kubeconfig.results.kubeconfig)"
+              - name: artifact-dir
+                value: "/workspace/artifacts/konflux-artifacts"
+          # validate that the cluster resources are available in another tekton step
+          - name: list-artifacts
+            onError: continue
+            image: quay.io/konflux-qe-incubator/konflux-qe-tools:latest
+            workingDir: "/workspace/"
+            script: |
+              #!/bin/bash
+              ls -la /workspace/
+              ls -la /workspace/artifacts/
+              ls -la /workspace/artifacts/gui_test_screenshots
+          - name: push-artifacts
+            onError: continue
+            ref:
+              resolver: git
+              params:
+                - name: url
+                  value: https://github.com/konflux-ci/tekton-integration-catalog.git
+                - name: revision
+                  value: main
+                - name: pathInRepo
+                  value: stepactions/secure-push-oci/0.1/secure-push-oci.yaml
+            params:
+              - name: workdir-path
+                value: "/workspace/artifacts"
+              - name: oci-ref
+                value: "quay.io/openshift-lightspeed/ols-console-artifacts:$(params.commit)"
+              - name: credentials-volume-name
+                value: ols-konflux-artifacts-bot-creds
+          - name: fail-if-any-step-failed
+            ref:
+              resolver: git
+              params:
+                - name: url
+                  value: https://github.com/konflux-ci/tekton-integration-catalog.git
+                - name: revision
+                  value: main
+                - name: pathInRepo
+                  value: stepactions/fail-if-any-step-failed/0.1/fail-if-any-step-failed.yaml
```

Comment:
@syedriko Why are we bringing this file back ? I split the console tests into pf5 and pf6 , below files  - right

Reply:
a rebase dragged it back in, I've now removed it. Thanks for catching!